src/NuGet.Core/NuGet.ProjectModel/ProjectLockFile/PackagesLockFile.cs

```diff
@@ -11,12 +11,22 @@ namespace NuGet.ProjectModel
 {
     public class PackagesLockFile : IEquatable<PackagesLockFile>
     {
-        public int Version { get; set; } = PackagesLockFileFormat.Version;
+        public int Version { get; set; }
 
         public string Path { get; set; }
 
         public IList<PackagesLockFileTarget> Targets { get; set; } = new List<PackagesLockFileTarget>();
 
+        public PackagesLockFile(int version)
+        {
+            Version = version;
+        }
+
+        public PackagesLockFile()
+        {
+            Version = PackagesLockFileFormat.Version;
```

Comment:
Given that we likely won't get to implement https://github.com/NuGet/Home/issues/9370 in the same release as this PR, we should use an incremented version only when CPVM is used. 
Otherwise regular users of lock files will get a needless version bump.

Reply:
The PackagesLockFileFormat.Version is 1 so no increase. 
https://github.com/NuGet/NuGet.Client/blob/c25b8fee2be5608c60bd712ff905985e8aebf59d/src/NuGet.Core/NuGet.ProjectModel/ProjectLockFile/PackagesLockFileFormat.cs#L18